### ISSUE TITLE
small simple site header

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-website#195]
## Why
* screen real estate, e.g., when have filters (especially for large text, small devices, etc.)

## For What

probably use for:
* [quickref](https://www.w3.org/WAI/WCAG21/quickref/)
* [tools list](https://www.w3.org/WAI/ER/tools/)
* [BAD](https://www.w3.org/WAI/demos/bad/)
* [WCAG-EM report tool](https://www.w3.org/WAI/eval/report-tool/#/)
* [ATAG Report Tool](https://atag-report-tool.hdv.now.sh/)

maybe:
* WCAG Understanding & Techniques pages
* "supplemental guidance"
* [laws & policies](https://www.w3.org/WAI/policies/)

probably not for:
* Tutorials - @yatil is integrating into site nav ([draft](https://wai-tutorials.netlify.com/tutorials/))

Hidde says:
> I envision this to be good for resources:
> - that focus more on a task than on getting information / are interactive (i.e. have filters)
> - that could be seen as stand alone 	
> - that don't fit well in the regular WAI navigation
> - that are one thing (ie don't require skip links, as they start with main content)
> - where having full navigation would distract more than it would help
> - require custom design

**Comment by hidde** [comment]
I have done a proposal for this in https://github.com/w3c/wai-atag-report-tool/issues/23#issuecomment-585791436, which is similar to what the [updated WCAG EM Report Tool](https://www.w3.org/WAI/eval/report-tool/) went live with recently.

**Comment by hidde** [comment]
(Hi @slhenry, I edited your comment to include a link to the draft, and turned the mention of @yatil into a mention)

**Comment by slhenry** [comment]
_**Note:** With the [ATAG Report Tool](https://atag-report-tool.hdv.now.sh/), we are working on an overall tool design that we plan to use as the new template design. Eventually we would like to update the design of the other tools listed above._

In looking at the [ATAG Report Tool](https://atag-report-tool.hdv.now.sh/) and thinking about [where "ATAG Report Tool" should go on each page](https://github.com/w3c/wai-atag-report-tool/issues/68#issuecomment-622426451), I came back to this broader issue -- and looked at all the other tools linked above.

All other tools have name top left and logos top right, except WCAG-EM Report Tool **and importantly, it is missing "WCAG-EM Report Tool" on all pages** except the first page.

I'm thinking:
* fairly important to have the Tool Name prominent on each page
* probably less important to have logos top left -- although consistency with main site is good
* screen real estate is important for several of these, which have lots of data per page including filters

For these reasons, I'd like us to revisit a design that has the name/title top left and the logos right.

**Comment by hidde** [comment]
> screen real estate is important for several of these, which have lots of data per page including filters 

Agreed, I think because of that, this header should be as small as possible, while addressing the need: “I want to find out where I am” (in a tool that is part of WAI).

> fairly important to have the Tool Name prominent on each page
> probably less important to have logos top left -- although consistency with main site is good 

Agreed with both. 

What about something like this::

<img width="1106" alt="" src="https://user-images.githubusercontent.com/178782/83042440-f6abe400-a041-11ea-9a52-22124a618c11.png">

I think what would be a really useful thing as part of this, is _some way to demonstrate how it fits into WAI_.

I don't know how we could best do this, maybe using some kind of expand/collapse. or by displaying the most important bit of the 'breadcrumb', for instance, for the ATAG Report Tool, it 'belongs' to WAI's work in Standards and Guidelines, specifically the standard for Authoring Tools.

Brainstorm: 

<img width="1126" alt="" src="https://user-images.githubusercontent.com/178782/83042952-85206580-a042-11ea-9ab4-b6fbbee54f3d.png">

Maybe one way to do this, is to define _one_ page/thing that a tool belongs to, for each tool. We could then add a link in the header to that page, so that people can read more about the broader picture for the specific tool. For ART, that could be [ATAG Overview](https://www.w3.org/WAI/standards-guidelines/atag/).

**Comment by slhenry** [comment]
+1 to the first one.

(Hidde & Shawn discussed the second point, and agree not needed -- 'cause users focused on tool & context is usually given e.g., in about this tool...)

**Comment by slhenry** [comment]
Hidde, when you get to it, please add this to the documents / components. Thanks!

(then close this issue)

**Comment by hidde** [comment]
This header has been added to Design Components as the [Tool Header component](https://wai-website-theme.netlify.app/components/tool-header/). 

I envision we could have other 'Tool X' components, like 'Tool Navigation'. That work is happening in https://github.com/w3c/wai-atag-report-tool/issues/152.
